actionpack/test/dispatch/request_test.rb

```diff
@@ -1061,7 +1061,7 @@ class RequestParameterFilter < BaseRequestTest
 
       filter_words << 'blah'
       filter_words << lambda { |key, value|
-        value.reverse! if key =~ /bargain/
```

Comment:
It's very difficult to judge which is better to warn or not if returned value is `nil`.
When a develop want to filtered value to be changed to `nil`, warnings are noisy. But I the possibility of these use cases are low.
So I agree to warn if the returned value is `nil`, or not String, because before this commit developers can change `value` to be only a String.